rsocket/framing/FrameTransport.cpp

```diff
@@ -164,8 +143,6 @@ void FrameTransport::cancel() {
 }
 
 void FrameTransport::outputFrameOrDrop(std::unique_ptr<folly::IOBuf> frame) {
```

Comment:
This class doesn't know if dropping the frame is a critical error or not, but a higher logic might wish to take action if frames get dropped. 
Might it be beneficial to return a bool, stating if the frame is dropped or processed?